tests/acceptance/settings-page/billing-test.js

```diff
@@ -0,0 +1,152 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { signIn, signInAsSubscriber, signInAsVipUser } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+import billingPage from 'codecrafters-frontend/tests/pages/settings/billing-page';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupWindowMock } from 'ember-window-mock/test-support';
+import percySnapshot from '@percy/ember';
+
+module('Acceptance | settings-page | billing-test', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupWindowMock(hooks);
+
+  test('membership section shows correct plan for subscriber with active subscription', async function (assert) {
+    testScenario(this.server);
+    signInAsSubscriber(this.owner, this.server);
+    const subscription = this.server.schema.subscriptions.first();
+    subscription.update('pricingPlanName', 'Yearly Plan');
+
+    await billingPage.visit();
+
+    assert.ok(billingPage.membershipSection.isVisible, 'membership section is visible');
+    assert.ok(billingPage.membershipSection.text.includes('Membership active'), 'shows active plan');
```

Comment:
@Arpan-206 I'd also test the exact text for when a membership expires (or when VIP access expires) - copy with "logic" is an easy place to have mistakes. It's okay to not test simple sentences but anything with interpolation / logic is worth testing

Reply:
An important case that we aren't testing: when a user has a membership and VIP access: (a) if VIP expires after membership we should mention it (b) if VIP expires before membership we should not mention it